output/github-pandoc.css

```diff
@@ -77,13 +77,21 @@ body {
     margin: 0;
 }
 
+/* 
+ * Page margins when printing
+ * https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2982#issuecomment-308714479
```

Comment:
Good catch! Accidentally included from a copy and paste. Fixed in https://github.com/greenelab/manubot-rootstock/pull/39/commits/3ee2d67b82d99e2cb0dc41ecb87e5a8522a67dcb